docker/Dockerfile-dev

```
@@ -0,0 +1,11 @@
+FROM golang:1.13.0-alpine
```

If `golang` version if updated in the `Dockerfile`, this should be updated too :smile: 

Can't we update this? :smile: 

```suggestion
FROM golang:1.14-alpine
```

---

Done!